### PR TITLE
3 example story 1

### LINKED ISSUE #3: Example Story 1
# Documentation of public methods
As a developer, I want public methods to be documented so that I can understand code written by others more easily.

## Acceptance Criteria
- [ ] There is documentation for each public method
- [ ] The documentation describes what the methods do
- [ ] The documentation is written in english
- [ ] The documentation has correct grammar

## DoD
- [ ] criteria 1

## Dependencies
- #2

### PR DESCRIPTION
**Pull request by martinjonsson01**
Closes #3.

# DoD
- [x] criteria 1
- [x] criteria 1
- [x] criteria 1
- [x] criteria 1

### CODE REVIEW

File: documents/technical-guide.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Technical Guide
+Here you will find links to different documentation sites and tutorials describing different aspects of this project.
+
+Use this document as guidance whenever you encounter something that is new to you.
+
+## Agile & Scrum
+* [The scrum guide](https://scrumguides.org/docs/scrumguide/v2020/2020-Scrum-Guide-US.pdf)
+* [Google's code review guide](https://google.github.io/eng-practices/review/reviewer/)
+
+## C#
```
Comment (Gigset) [inline documents/technical-guide.md:10]: Why did you choose C? D is a better letter
 * Punktlista
 [ ] Box list

File: documents/technical-guide.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    * `// Arrange // Act // Assert`
+    * Important that `// Act` section is a single statement, so it only has one reason to fail
+  * Use built-in assertions instead of writing complex assertion logic that asserts multiple things in a row
+* Try to follow [TDD](https://www.codecademy.com/article/tdd-red-green-refactor)
+* Test-writing frameworks:
+  * [Unit tests](https://xunit.net/)
+  * [UI testing](https://docs.microsoft.com/en-us/aspnet/core/blazor/test?view=aspnetcore-6.0)
+  * [Fake data](https://github.com/bchavez/Bogus#the-great-c-example)
+  * [Assertion](https://fluentassertions.com/introduction)
+  * [Mocking](https://github.com/Moq/moq4/wiki/Quickstart)
```
Comment (Gigset) [inline documents/technical-guide.md:36]: THis is a multiline code :)

**Review by Gigset — CHANGES_REQUESTED** [review]
LGTM

- [ ]

**Review by Gigset — APPROVED** [review]
(y)

**Review by XwayTomy — APPROVED** [review]
GGWP

**Pull request merged**